connectors/src/connectors/notion/temporal/activities.ts

```diff
@@ -1812,6 +1812,8 @@ export async function renderAndUpsertPageFromCache({
               // We only update the rowId of for the page without truncating the rest of the table (incremental sync).
               truncate: false,
               parents,
+              title: parentDb.title ?? "Untitled Notion Database",
+              mimeType: "notion/database",
```

Comment:
nit: would go for "application/vnd.notion.database" but that's my conformist side :)

Reply:
sure ! What does `vnd` stand for here ?

Reply:
vendor :-)